language/stdlib/modules/libra_system.move

```diff
@@ -16,8 +16,7 @@ module LibraSystem {
     }
 
     struct ValidatorSetChangeEvent {
-        scheme: u8,
-        new_validator_set: vector<ValidatorInfo>,
+        last_reconfiguration_time: u64,
```

Comment:
Are we going to get rid of `last_reconfiguration_time` in `libra_system` in the PRs to come?

Reply:
yup